docs/ca-bundle-configuration.md

```diff
@@ -0,0 +1,282 @@
+# CA Bundle Configuration for LlamaStackDistribution
```

Comment:
We can just add a section in README with a reference to updated spec, similar to ConfigMap [changes](https://github.com/llamastack/llama-stack-k8s-operator?tab=readme-ov-file#using-a-configmap-for-runyaml-configuration). 

These docs(ca-bundle-configuration.md & vllm-tls-testing.md), although helpful seem to be more of helper documents. Maybe we can move them under `docs/additional` or `docs/guides` ?